lecture_1/server.js

```diff
@@ -0,0 +1,23 @@
+const http = require('http');
+
+http.createServer((request, response) => {
+  if (request.method === 'GET' && request.url === '/stats') {
+    const date = new Date();
+
+    const data = {
+      equesters_ip_address: request.socket.address().address,
+      current_GMT_time: `${date.toLocaleDateString()} ${date.toLocaleTimeString()}`,
+    };
+
+    response.statusCode = 200;
+    response.end(JSON.stringify(data));
+  } else if (request.method === 'POST' && request.url === '/users') {
```

Comment:
You can use immediate `return` statements to avoid retundant `else if` and `else`